### PR TITLE
server: update models to include case, patient and virus

### PR DESCRIPTION
**Pull request by u3556490**
Model updated to include case, patient and virus. Relationships set up accordingly.
Included in this pull request are documents that may be useful when we make the docs later.

### CODE REVIEW

**Review by uday1331 — APPROVED** [review]
LGTM!

File: server/hotzone/models.py [hunk 1]
```diff
@@ -9,3 +10,72 @@ def __str__(self):
...
+	# enum type 
+	class Origin(models.TextChoices):
+		LOCAL = 'L'
+		IMPORTED = 'I'
+
+	case_no = models.IntegerField()
+	confirmed = models.DateField()
+	origin = models.CharField(max_length=7, choices=Origin.choices, default=Origin.LOCAL)
+	patient = models.ForeignKey(Patient, on_delete=models.CASCADE)	
+	infected = models.ForeignKey(Virus, on_delete=models.CASCADE)
```
Comment (ahsan8244) [inline server/hotzone/models.py:61]: let's name this field "virus" for clarity

**Review by ahsan8244 — CHANGES_REQUESTED** [review]

File: server/hotzone/models.py (hunk 1 above)
Comment (ahsan8244) [reply server/hotzone/models.py:61]: No other issues from my side 👍

Comment (uday1331) [reply server/hotzone/models.py:61]: Oh good catch

**Comment by ahsan8244** [comment]
Maybe let's not push build folders? not a big issue though, what do you think @uday1331

**Comment by uday1331** [comment]
> Maybe let's not push build folders? not a big issue though, what do you think @uday1331

We need the build folder for the deployment. I think if we add it to `.gitignore`, it may ignore it when we push the subtree too.

**Comment by ahsan8244** [comment]
> > Maybe let's not push build folders? not a big issue though, what do you think @uday1331
> 
> 
> 
> We need the build folder for the deployment. I think if we add it to `.gitignore`, it may ignore it when we push the subtree too. 

Oh I see, alright

**Comment by uday1331** [comment]
> > > Maybe let's not push build folders? not a big issue though, what do you think @uday1331
> > 
> > 
> > We need the build folder for the deployment. I think if we add it to `.gitignore`, it may ignore it when we push the subtree too.
> 
> Oh I see, alright

Honestly, this was lazy on my part, should have figured out a better git flow. I just went with whatever came to mind. Didn't really research this.

File: server/hotzone/models.py (hunk 1 above)
Comment (u3556490) [reply server/hotzone/models.py:61]: ok line 61 `infected `-> `virus`

**Comment by u3556490** [comment]
Requesting to close pull request and delete relevant branch.
Changes made (on aggregate) would be pushed to a new one, because this branch would have one excess commit (build is included here but better not). New PR shall be opened accordingly.

**Pull request closed without merging**